compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -666,6 +666,48 @@ lambdaArgument lambdaArgumentFromSignedScalar(int64_t scalar) {
   return scalar_arg;
 }
 
+concreteprotocol::LweCiphertextEncryptionInfo::Reader
+clientParametersInputEncryptionAt(
+    ::concretelang::clientlib::ClientParameters &clientParameters,
+    size_t inputId, std::string circuitName) {
```

Comment:
just naming that more a position that an ID

Reply:
but that's the naming used in client parameters, isn't it?

Reply:
If I'm not wrong there was not identified, it's just a position on the list
```
struct CircuitEncodingInfo {
  # A circuit encodings is described by the set of encodings used for its inputs and outputs and its
  # name. This structure represents this circuit encoding signature.
  #
  # Note:
  #   The order of the input and output lists matters. The order of values should be the same when 
  #   executing the circuit. Also, the name is expected to be unique in the program.
       
  inputs @0 :List(EncodingInfo); # The ordered list of input encoding infos.
  outputs @1 :List(EncodingInfo); # The ordered list of output encoding infos.
  name @2 :Text; # The name of the circuit.
}
```